src/LightKrylov.fypp

```diff
@@ -57,6 +63,12 @@ module LightKrylov
     public :: abstract_hermitian_linop_${type[0]}$${kind}$
     #:endif
     #:endfor
+
+    ! AbstractSystems exports.
+    #:for kind in REAL_KINDS
+    public :: abstract_system_r${kind}$
+    public :: abstract_jacobian_linop_r${kind}$
+    #:endfor
```

Comment:
No complex versions?

Reply:
I did not add them because I did not write any tests for them.

I'll add them and we can see.